spec/game_spec.rb

```diff
@@ -0,0 +1,116 @@
+require './lib/game.rb'
+
```

Comment:
very extensive testing, covered all the scenarios required in order for the game to be played.